frappe/custom/doctype/custom_field/custom_field.py

```diff
@@ -46,6 +46,9 @@ def validate(self):
 		if not self.fieldname:
 			frappe.throw(_("Fieldname not set for Custom Field"))
 
+		if self.fieldname in fieldnames:
+			frappe.throw(_("A field with the name '{}' already exists in doctype {}.".format(self.fieldname, self.dt)))
```

Comment:
The format can (and should) be placed after the translation syntax:

```suggestion
			frappe.throw(_("A field with the name '{}' already exists in doctype {}.").format(self.fieldname, self.dt))
```

That way you can plug in data after translating the string. If you place `format()` inside the translation syntax, it tries to plug in the values before translating it and that is why it fails.